README.md

```diff
@@ -1,2 +1,54 @@
-# rtx-kg2-gateway
+# RTX-KG2 Gateway
+
 Enabling RTX-KG2 data access through various means.
+
+## Installation
+
+Please use Python [`poetry`](https://python-poetry.org/) to run and install related content.
+The Poetry environment for this project includes dependencies which help run IDE environments, manage the data, and run workflows.
+
+```bash
+# after installing poetry, create the environment
```

Comment:
Saying something about where to run the command -- in this case, I assume in the repo root -- could be helpful. (I've seen people, e.g., clone a repo and then just start running the commands from the repo's README in the folder where they cloned the repo, not in the repo folder itself, so perhaps a gentle reminder would help.)

Reply:
Good idea, I'll add in an additional note here!